ananse/peakpredictor.py

```diff
@@ -37,29 +39,32 @@
 class PeakPredictor:
     atac_data = None
     histone_data = None
+    cage_data = None
     factor_models = {}
 
     def __init__(
-        self,
-        reference=None,
-        atac_bams=None,
-        histone_bams=None,
-        columns=None,
-        regions=None,
-        genome="hg38",
-        pfmfile=None,
-        factors=None,
-        pfmscorefile=None,
-        ncore=4,
+            self,
+            reference=None,
+            atac_bams=None,
+            histone_bams=None,
+            cage_tpms=None,
+            columns=None,
+            regions=None,
+            genome="hg38",
+            pfmfile=None,
+            factors=None,
+            pfmscorefile=None,
+            ncore=4,
     ):
         if reference is None:
             reference = os.path.join(PACKAGE_DIR, "db", "default_reference")
         if not os.path.exists(reference):
             raise NotADirectoryError(f"Could not find {reference}")
         self.data_dir = reference
 
-        if atac_bams is None and histone_bams is None:
-            raise ValueError("Need either ATAC-seq or H3K27ac BAM file(s).")
+        if atac_bams is None and histone_bams is None and cage_tpms is None:
+            raise ValueError("Need either ATAC-seq or H3K27ac BAM file(s) "
+                             "or CAGE bidirectional sites (TPM) .tsv file.")
```

Comment:
This message is outdated :o 

"Need either ATAC-seq or H3K27ac BAM file(s), ATAC-seq or H3K27ac coverage table(s), or a CAGE bidirectional sites (TPM) .tsv file."